### DISCUSSION TITLE
CS8034 C# Unable to load Analyzer assembly {0}: {1}

### DISCUSSION THREAD

**Discussion by simonbrazer** [codewars/codewars.com#2453] [Q&A] [answered]
![problem](https://user-images.githubusercontent.com/88495025/129254496-3eedab68-2741-489c-8dad-5de9b57d97a3.png)
Hello.I'm trying to download these Packages from this video https://www.youtube.com/watch?v=C5cnZ-gZy2I timeframe in video 1:06:29 and only 1/4 installs succesfully,others fail.Tried seeing the problem in Github,but had no idea what people are talking about.Maybe someone could help?

**Comment by DonaldKellett** [comment, marked as answer]
Codewars Discussions on GitHub are not meant for general programming help, only
for discussing stuff related to Codewars: https://www.codewars.com

**Reply by simonbrazer** [reply]
It doesn't let me post anything and nobody is able to help me.
